README.md

```diff
@@ -30,19 +30,31 @@ docker run --platform linux/amd64 --publish 8000:8080 ghcr.io/mermaid-js/mermaid
 ### To configure renderer URL
 
 When building set the MERMAID_RENDERER_URL build argument to the rendering service.
-Default is `https://mermaid.ink`
+Default is empty disabling PNG and SVG links under Actions
 
 ### To configure Kroki Instance URL
 
 When building set the MERMAID_KROKI_RENDERER_URL build argument to your Kroki instance.
-Default is `https://kroki.io`
+Default is empty disabling Kroki link under Actions
```

Comment:
We should set mermaid.ink and kroki.io as examples here (not default), so users can know which services to use.

Reply:
I will set them to default so that the e2e test can check the functionality as it's doing already - so there's no regression.